.github/workflows/deploy-production.yml

```diff
@@ -15,13 +16,13 @@ jobs:
 
   # up to date scan of the staging instance
   scan-staging:
-    name: Zap scan of the staging site
+    name: Zap Scan
```

Comment:
It seems important to note that, somewhat counterintuitively, the ZAP scan is of the staging site even though this is the production deploy. So maybe leave this description, or add a comment as to the fact that it's not a mistake/oversight...? Also there is another "Zap Scan" step below that is otherwise undifferentiated when you look at the list of steps in this workflow.

Reply:
added verbage back